mention-bot.js

```diff
@@ -9,7 +9,17 @@
  * @flow
  */
 
-var downloadFileSync = require('download-file-sync');
+var githubWebClient = require('./githubWebClient');
+
+var downloadFileSync = function(url, cookies) {
+  var command = 'curl -L ' + url;
```

Comment:
please use an array here, otherwise it's not going to be escaped and you can basically run anything if the url or cookies is compromised